test/parallel/test-child-process-exec-encoding.js

```diff
@@ -1,17 +1,17 @@
 'use strict';
-const common = require('../common');
```

Comment:
Same here. Can we move the child process code to a fixture?

Reply:
I think moving these things to a fixture is the wrong thing to do. We have a lot of fixtures and there is no benefit this in a fixture as far as I see it. It is much easier to handle this test if everything is together. The fixtures folder is full with things of which probably no one has an idea about what is really in there.